dencam/networking.py

```diff
@@ -41,3 +41,30 @@ def get_network_info():
             text += (ssid.decode('utf-8').split('ESSID:')[-1].replace('"', ''))
 
     return text
+
+
+class AirplaneMode:
+
+    def __init__(self, configs):
+        apm = configs['AIRPLANE_MODE']
+        if apm is True:
+            self.ap_mode_on()
+        if apm is False:
+            self.ap_mode_off()
+        self.enabled = apm
+
+    def ap_mode_off(self):
+        subprocess.call("rfkill unblock wlan", shell=True)
```

Comment:
Good that we have switched to subprocess module.  You might want to read up on the security issue associated with setting ```shell=True```.  We're not going to worry about this for now.